### ISSUE TITLE
ffmpeg not available under Ubuntu 14.04

### ISSUE THREAD

**Issue by ZdenekM** [Zulko/moviepy#189]
I'm getting this error
```IOError: Unable to download 'ffmpeg.linux64'. Perhaps there is a no internet connection? If there is, please report this problem.```
probably because ffmpeg is not in Ubuntu repositories anymore... Would be nice if moviepy can use libav-tools instead - fork of ffmpeg which is shipped with Ubuntu.

**Comment by Zulko** [comment]
I have used MoviePy on Ubuntu 12 and 14 on 32- and 64-bit systems and never had this problem so I guess you have an issue somewhere else.

MoviePy does not use the ffmpeg from the Ubuntu repos (even in Ubuntu 12 the ffmpeg in the repos was old and deprecated), what happens is that ImageIO downloads a binary of ffmpeg from its own repo. Apparently imageio had a problem downloading ffmpeg.linux64 in your case.

Can you retry in a few moments and tell me if the problem persists, at which case it may be worth opening an issue in the ImageIO project.

Cheers

**Comment by ZdenekM** [comment]
Ouch, I'm sorry. I thought it uses ffmpeg from repos... Well, now I see. Now it works - probably some issue on server. Thanks and once again sorry for noise :-)

**Comment by nemetniki** [comment]
Hi! I have the same issue in Canopy. Do you have any suggestions?
Thanks

**Comment by Zulko** [comment]
Make sure you are connected to the web and retry in a few moments. Let us know if you still have the problem.

**Comment by leoncss** [comment]
Hi Zulko,

I'm using Ubuntu 14.04 and I tried several times to download ffmpeg.linux64 but i haven't been successful. I am very sure that I am connected to the web. 

I tried downloading the executable file from https://github.com/imageio/imageio-binaries/raw/master/ffmpeg/ffmpeg.linux64 but I do not know where to place it, or how to use it. Also, I downloaded your ffmpeg.py file and tried to run it but I got a ValueError: Attempted relative import in non-package. 

I am quite lost and don't know what to do. Any help or advice will be greatly appreciated!

Thank you
